### PR TITLE
Current page is not added to link provider dropdown

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
Fliplet/fliplet-studio#4498

## Description
Added omit page property.

## Screenshots/screencasts
Current page is not displayed.
primary button demo

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+data.omitPages = page ? [page.id] : [];
 
 var linkActionProvider = Fliplet.Widget.open('com.fliplet.link', {
   // If provided, the iframe will be appended here,
   // otherwise will be displayed as a full-size iframe overlay
   selector: '#action',
   // Also send the data I have locally, so that
   // the interface gets repopulated with the same stuff
-  data: data.action,
+  data: data,
```
Comment (tonytlwu) [inline js/interface.js]: Why is this changed? This looks like it'll affect how current widget instances load configurations.

Comment (MaksymShokin) [reply js/interface.js]: @tonytlwu  Fixed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Medium
Affected: Primary button configurations
Verify: Configurations are correctly stored and restored
